### ISSUE TITLE
Error helpers should be part of the SDK's

### ISSUE THREAD

**Issue by wtrocki** [redhat-developer/app-services-sdk-go#211]
https://github.com/redhat-developer/app-services-cli/blob/srsdata/pkg/api/srs/error.go#L97-L122

**Comment by craicoverflow** [comment]
Will that be used anywhere other than the CLI?

**Comment by wtrocki** [comment]
yes.. Get API error implementation is tricky that involves casting errors and without that sdk users cannot get full error information nor handle it. Obviously status codes and error model will be different for each SDK but they should be there.
